deploy/README.md

```diff
@@ -45,27 +45,28 @@ The diagram below illustrates the components of the Kubernetes collection soluti
 
 ## Minimum Requirements
 
-Name | Version
--------- | -----
-K8s | 1.16+
-Helm | 3.4+
+| Name | Version |
+|------|:-------:|
+| K8s  |  1.16+  |
+| Helm |  3.4+   |
 
 ## Support Matrix
 
 The following table displays the tested Kubernetes and Helm versions.
 
-Name | Version
--------- | -----
-K8s with EKS | 1.16<br/>1.17<br/>1.18
-K8s with Kops | 1.16<br/>1.17<br/>1.18
-K8s with GKE | 1.16
-K8s with AKS | 1.16<br/>1.17<br/>1.18
-OpenShift | 4.6
-Helm | 3.4.0 (Linux)
-kubectl | 1.16.0
+| Name          |        Version         |
+|---------------|:----------------------:|
+| K8s with EKS  | 1.16<br/>1.17<br/>1.18 |
+| K8s with Kops | 1.16<br/>1.17<br/>1.18 |
+| K8s with GKE  |          1.16          |
+| K8s with AKS  | 1.16<br/>1.17<br/>1.18 |
+| OpenShift     |          4.6           |
+| Helm          |     3.4.0 (Linux)      |
+| kubectl       |         1.16.0         |
```

Comment:
It doesn't look like very readable with center alignment

Reply:
I don't agree but I can revert this to please the crowd.